app/services/sufia/admin_set_create_service.rb

```diff
@@ -7,7 +7,7 @@ def self.create_default!
       return if AdminSet.exists?(DEFAULT_ID)
       admin_set = AdminSet.new(id: DEFAULT_ID, title: ['Default Admin Set'])
       begin
-        new(admin_set, nil).create
+        new(admin_set, nil, 'default').create
```

Comment:
I'm not going to get hung up on this, but maybe a constant would call out this hardcoding.